stl/inc/chrono

```diff
@@ -601,6 +601,8 @@ namespace chrono {
         }
     };
 
+#pragma float_control(precise, on, push)

```

Comment:
Did you observe a significant codegen difference by setting these?

Reply:
Did not look at code at all.
But since it can be included into translation units with different `/fp` switch, I want to be safe.

Reply:
(And anyway I think more important is conclusion on #656 , it may change the direction of this PR)

Reply:
I spot the difference under /arch:IA32 on x86 : https://godbolt.org/z/nbqynW

Reply:
Though I cannot explain this difference, I mean I don't see why `/fp:precise` version would be more precise.